### PR TITLE
updated hotline sdk to 1.2.1

### PR DESCRIPTION
**Pull request by sksethi25**
updated sdk to 1.2.1
added build/index.js to dynamincally change strings.xml

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,8 +9,13 @@
       "hotline": "js"
     },
     "extensions": {
+      "build": "build",
```
Comment (rampr) [inline package.json:12]: please add build folder

File: android/strings.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string name="hotline_file_provider_authority">com.hashcube.test</string>
+</resources>
```
Comment (bijoshtj) [inline android/strings.xml]: add newline

File: package.json (hunk 1 above)
Comment (sksethi25) [reply package.json:12]: added

File: android/strings.xml (hunk 2 above)
Comment (sksethi25) [reply android/strings.xml]: added

**Review by rampr — APPROVED** [review]

File: build/index.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    strings_file = 'hotline_strings.xml',
+    conf;
+
+  if (config.target == 'native-android') {
+    conf = {
+      hotline_file_provider_authority: packageName + ".provider",
+    };
+    xmlStr = new xmldoc.XmlDocument(fs.readFileSync(path.join(__dirname, '../android/strings.xml'), 'utf-8'));
+
+    for (var i = 0; i < xmlStr.children.length; i++) {
```
Comment (bijoshtj) [inline build/index.js]: since theres only one property, no for loop needed.

**Review by rampr — APPROVED** [review]

**Review by bijoshtj — APPROVED** [review]

**Pull request merged**